### ISSUE TITLE
"Cannot get property 'apiLevel' on null object" thrown when project doesn't set targetSdkVersion

### ISSUE THREAD

**Issue by jkasten2** [OneSignal/OneSignal-Gradle-Plugin#20]
Android projects don't require `targetSdkVersion` to be set. However this plugin does not handle this correctly which ends up throwing the following error.
```java
Could not resolve com.onesignal:OneSignal:[3.7.1, 3.99.99].
Required by:
    project :app
 > Cannot get property 'apiLevel' on null object
```

The following line that need to be updated.
https://github.com/OneSignal/OneSignal-Gradle-Plugin/blob/0.8.1/src/main/groovy/com/onesignal/androidsdk/GradleProjectPlugin.groovy#L166

**Comment by alexnivanov** [comment]
@jkasten2 thank you so much man, I was ready to break my head on this message 👍

**Comment by jkasten2** [comment]
Fixed in PR above, will be included in the next release

**Comment by jkasten2** [comment]
@alexnivanov This fix is now available in the [0.8.2 Release](https://github.com/OneSignal/OneSignal-Gradle-Plugin/releases/tag/0.8.2)
